modules/ossm-rn-new-features.adoc

```diff
@@ -55,28 +55,28 @@ This release adds documentation for migrating from a multitenant mesh to a clust
 
 === {SMProductName} Operator on ARM-based clusters
 
-This release provides the {SMProductName} Operator on ARM-based clusters as a generally available feature. 
+This release provides the {SMProductName} Operator on ARM-based clusters as a generally available feature.
 
 === Integration with {DTProductName} (Tempo) Stack
 
 This release introduces a generally available integration of the tracing extension provider(s). You can expose tracing data to the {DTProductName} (Tempo) stack by appending a named element and the `zipkin` provider to the `spec.meshConfig.extensionProviders` specification. Then, a telemetry custom resource configures Istio proxies to collect trace spans and send them to the Tempo distributor service endpoint.
 
 [NOTE]
 ====
-{DTProductName} (Tempo) Stack is not supported on {ibm-z-title}. 
+{DTProductName} (Tempo) Stack is not supported on {ibm-z-title}.
 ====
 
 === OpenShift Service Mesh Console plugin
 
-This release introduces a generally available version of the OpenShift {SMProductShortName} Console (OSSMC) plugin. 
+This release introduces a generally available version of the OpenShift {SMProductShortName} Console (OSSMC) plugin.
 
-The OSSMC plugin is an extension to the OpenShift Console that provides visibility into your Service Mesh. With the OSSMC plugin installed, a new Service Mesh menu option is available in the left-hand navigation of the web console, as well as new Service Mesh tabs that enhance existing Workloads and Service console pages. 
+The OSSMC plugin is an extension to the OpenShift Console that provides visibility into your Service Mesh. With the OSSMC plugin installed, a new Service Mesh menu option is available on the left side of the web console, as well as new Service Mesh tabs that enhance existing Workloads and Service console pages.
```

Comment:
We should revise this, based on this role for "navigation" panes: https://redhat-documentation.github.io/supplementary-style-guide/#accessibility-visual-info

maybethis, or something like it:

```suggestion
The OSSMC plugin is an extension to the OpenShift Console that provides visibility into your Service Mesh. With the OSSMC plugin installed, a new Service Mesh menu option is available on the navigation pane of the web console, as well as new Service Mesh tabs that enhance existing Workloads and Service console pages.
```